code/_onclick/click.dm

```diff
@@ -16,6 +16,28 @@
 	Note that this proc can be overridden, and is in the case of screen objects.
 */
 
+/client/Click(var/atom/target, location, control, params)
+//	if(world.time <= next_click) // Hard check, before anything else, to avoid crashing
+//		return
+
+//	next_click = world.time + 1
+
+//	if(buildmode && !istype(target, /obj/screen))
+//		buildmode.build_click(src.mob, params, target)
```

Comment:
Why is this block commented out? did you remove build mode ?

Reply:
That code is my, but i made it for other (dead) server, i just recode it. In eris no have procs for client, so i just remove not compable things